pnpm-lock.yaml

```diff
@@ -42561,7 +42682,7 @@ packages:
     dependencies:
       react: 17.0.2
       react-dom: 17.0.2(react@17.0.2)
-    dev: true
+    dev: false
```

Comment:
How come this is being set to false?

Reply:
It's probably because when `re-resizable` was a dependency of `devtools-view`, it was a dev-dependency. Now in `devtools-example` it's a regular dependency, because `devtools-example` as a whole is intended to be just an example, not a package to be published and consumed by others like `devtools-view`, so it's not an issue that it uses regular dependencies to be able to showcase what it wants to.

Reply:
Yup!